commands/application.angular.js

```diff
@@ -1,10 +1,16 @@
 const path = require('path');
-const runNpxCommand = require('../utility/run-npx-command');
+const fs = require('fs');
+const runCommand = require('../utility/run-command');
 
 function runSchematicCommand(schematicCommand, options, evaluatingOptions) {
     const collectionName = 'devextreme-schematics';
+    const packageJsonPath = path.join(process.cwd(), `node_modules/${collectionName}/package.json`)
```

Comment:
The variable name does not tell `package.json` file of which package you are checking.